x-pack/plugins/ingest_pipelines/public/application/lib/utils.ts

```diff
@@ -5,21 +5,25 @@
  * 2.0.
  */
 
-export const stringifyJson = (json: any): string =>
-  Array.isArray(json) ? JSON.stringify(json, null, 2) : '[\n\n]';
+export const stringifyJson = (json: any, isList: boolean = true): string =>
```

Comment:
can we add tests for this?

Reply:
Sorry, I didn't notice that there is a file that tests these functions. I added a few more test cases and actually refactored the `stringifyJson` function so that it works correctly when it is an argument that is not of key-value object type.